lib/sqlalchemy/dialects/mssql/base.py

```diff
@@ -2057,6 +2057,9 @@ def visit_length_func(self, fn, **kw):
     def visit_char_length_func(self, fn, **kw):
         return "LEN%s" % self.function_argspec(fn, **kw)
 
+    def visit_aggregate_strings_func(self, fn, **kw):
+        return "string_agg%s" % self.function_argspec(fn, **kw)
+
     def visit_concat_op_expression_clauselist(
```

Comment:
**mike bayer** (zzzeek) wrote:

aggregate_strings(), but we should just give the function a constructor

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4781

Reply:
**Federico Caselli** (CaselIT) wrote:

added init so this is no longer needed

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4781